### PR TITLE
[incubator/gitea] Initial commit of chart.

### PR DESCRIPTION
**Pull request by jpds**

### CODE REVIEW

File: incubator/gitea/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fullname" . }}-config
```
Comment (unguiculus) [inline incubator/gitea/templates/configmap.yaml:5]: `app` label should be `{{ template "name" . }}` on all resources.

File: incubator/gitea/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: {{ default "gitea" .Values.service.nameOverride }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "fullname" . }}-config
```
Comment (unguiculus) [inline incubator/gitea/templates/configmap.yaml:10]: All other resources are named `{{ template "fullname" . }}`. So should be the configmap.

File: incubator/gitea/templates/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,128 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fullname" . }}-config
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: {{ default "gitea" .Values.service.nameOverride }}
```
Comment (unguiculus) [inline incubator/gitea/templates/configmap.yaml:7]: I don't see why a `component` label on the configmap would make sense. I'd suggest you remove it.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Note that we've started namespacing templates (see #1785). You should apply this to your chart (can be done in another PR) in order for this chart to move to stable.

**Comment by unguiculus** [comment]
Marking this a stale. Please update within one week.

**Comment by viglesiasce** [comment]
Closing as stale.

**Pull request closed without merging**